### PR TITLE
Shrink sizeof Value struct

### PR DESCRIPTION
**Pull request by dutor**
Depends on vesoft-inc/nebula-common#549 
Required by vesoft-inc/nebula-graph#1094

### CODE REVIEW

**Review by critical27 — APPROVED** [review]
LGTM

File: src/codec/RowWriterV2.cpp [hunk 1]
```diff
@@ -780,21 +780,28 @@ WriteResult RowWriterV2::write(ssize_t index, const Time& v) noexcept {
 WriteResult RowWriterV2::write(ssize_t index, const DateTime& v) noexcept {
     auto field = schema_->field(index);
     auto offset = headerLen_ + numNullBytes_ + field->offset();
+    int16_t year = v.year;
```
Comment (Shylock-Hg) [inline src/codec/RowWriterV2.cpp:783]: Maybe we could store the bit field directly in next incompatible version.

**Review by Shylock-Hg — APPROVED** [review]

File: src/codec/RowWriterV2.cpp (hunk 1 above)
Comment (dutor) [reply src/codec/RowWriterV2.cpp:783]: It might have trouble with the secondary index, but we could have a try.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**